test/invariant/fuzz/handlers/HandlerAllo.t.sol

```diff
@@ -33,65 +34,299 @@ contract HandlerAllo is Setup {
+                ghost_poolManagers[_poolId].push(_managers[_i]);
+            }
+        }
+    }
+
+    function handler_removePoolManagers(uint256 _idSeed) public onlyValidPoolAndAdmin(_idSeed) {
+        uint256 _poolId = _getPoolId(_idSeed);
+        address[] memory _managers = ghost_poolManagers[_poolId];
+
+        // Remove pool managers - will revert if caller is not a pool admin of the pool id
+        (bool _succ,) =
+            targetCall(address(allo), 0, abi.encodeWithSelector(IAllo.removePoolManagers.selector, _poolId, _managers));
+
+        if (_succ) {
+            delete ghost_poolManagers[_poolId];
+        }
+    }
+
+    function handler_addPoolManagersInMultiplePools(uint256[] calldata _seeds, address[] calldata _managers)
+        public
+        onlyValidPoolsAdmin(_seeds)
+    {
+        uint256[] memory _poolIds = _getPoolsIds(_seeds);
+
+        // Add pool managers in multiple pools - will revert if caller is not a pool admin of any pool id
+        (bool _succ,) = targetCall(
+            address(allo), 0, abi.encodeWithSelector(IAllo.addPoolManagersInMultiplePools.selector, _poolIds, _managers)
+        );
+
+        if (_succ) {
+            for (uint256 _i; _i < _poolIds.length; ++_i) {
+                uint256 _poolId = _poolIds[_i];
+                for (uint256 _j; _j < _managers.length; ++_j) {
+                    ghost_poolManagers[_poolId].push(_managers[_j]);
+                }
+            }
+        }
+    }
+
+    function handler_removePoolManagersInMultiplePools(uint256[] calldata _seeds, address[] calldata _managers)
+        public
+        onlyValidPoolsAdmin(_seeds)
+    {
+        uint256[] memory _poolIds = _getPoolsIds(_seeds);
+
+        // Remove pool managers in multiple pools - will revert if caller is not a pool admin of any pool id
+        (bool _succ,) = targetCall(
             address(allo),
             0,
-            abi.encodeWithSelector(
-                IAllo.updatePoolMetadata.selector,
-                poolId,
-                metadata
-            )
+            abi.encodeWithSelector(IAllo.removePoolManagersInMultiplePools.selector, _poolIds, _managers)
         );
+
+        if (_succ) {
+            for (uint256 _i; _i < _poolIds.length; ++_i) {
+                uint256 _poolId = _poolIds[_i];
+                delete ghost_poolManagers[_poolId];
+            }
+        }
     }
 
-    function handler_updatePercentFee(uint256 _newPercentFee) public {
-        _newPercentFee = bound(_newPercentFee, 0, 1e18);
+    function handler_recoverFunds(address _token, address _recipient) public {
+        // Recover funds - will revert if caller is not the owner
+        targetCall(address(allo), 0, abi.encodeWithSelector(IAllo.recoverFunds.selector, _token, _recipient));
+    }
 
-        // Update the percent fee - will revert if wrong caller
+    function handler_registerRecipient(
+        uint256 _seed,
+        address[] memory _recipientAddresses,
```

Comment:
imo, we could/should use recipient in our actors, wdyt?

Reply:
good idea